lib/key_tools.ex

```diff
@@ -127,6 +127,7 @@ defmodule KeyTools do
   iex(2)> stringify_keys [%{atom_key: %{42 => [%{another_key: 23}]}}]
   [%{"atom_key" => %{"42" => [%{"another_key" => 23}]}}]
   """
+  def stringify_keys(%{__struct__: _any}=struct), do: struct
```

Comment:
I'd prefer just `_` over `_any` as we've already used it elsewhere in this file.